### ISSUE TITLE
Incorrect rendering of bounding rect and convex hull for US layer

### ISSUE THREAD

**Issue by skazemi** [frewsxcv/rgis#113]
Bounding rectangle and convex hull are rendered incorrectly when the layer (US) passed beyond the (-180, 180) edges
It's related to the small part of US being on the right side of screen (-188.9)

**Comment by frewsxcv** [comment]
Yeah the -188.9 becomes a positive value upon transforming to EPSG:3857. I'm not sure how to handle this! If there's a way with PROJ to preserve the <-180 value. Or if that's the wrong approach. 🤔 

**EPSG:4326:**

<img width="1392" alt="Screenshot 2022-12-30 at 11 50 59 AM" src="https://user-images.githubusercontent.com/416575/210094254-1b2e5379-6398-4871-84d8-99a38683a383.png">

**EPSG:3857:**

<img width="1392" alt="Screenshot 2022-12-30 at 11 51 53 AM" src="https://user-images.githubusercontent.com/416575/210094269-809ac437-0252-4802-ba63-04c058a14f31.png">

**Comment by JmsPae** [comment]
Tried to do similar operations (convex hull, minimum bounding geometry) on the geojson in QGIS and it wasn't having it (invalid geometry). Paragraph 3.1.9 of the [geojson standard](https://www.rfc-editor.org/rfc/rfc7946) states

> In representing Features that cross the antimeridian,
   interoperability is improved by modifying their geometry.  Any
   geometry that crosses the antimeridian SHOULD be represented by
   cutting it in two such that neither part's representation crosses the
   antimeridian. 

So I suppose the data source isn't really up to standard, but the language isn't exactly absolute. Other data sources for US states also don't go past the antimeridian (-180 lon).

If you want to account for this kind of geometry anyway, I'd imagine you'd have to clip anything going past the antimeridian and move it to the other side of things. But I don't think there's any shame in just choosing another data source that doesn't.

I suppose you can try to make a pull request on the [original geojson](https://github.com/PublicaMundi/MappingAPI/blob/master/data/geojson/us-states.json), but it doesn't seem like the author is active.
